<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>903</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1121</width>
      <height>881</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="lineWidth">
          <number>3</number>
         </property>
         <property name="midLineWidth">
          <number>3</number>
         </property>
         <property name="text">
          <string>1
2
3
4
5
</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="minimumSize">
          <size>
           <width>620</width>
           <height>0</height>
          </size>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
         <property name="plainText">
          <string>{
  int k; int sum, input, i;
  sum = 0;
  for i=0; to ((3^(((144))))) by ((k+(1))) while( sum &lt; 1000)
	{
		read(input);
		if (input &gt; 0)
	 	{
			sum = sum + input;
			write(sum);
		}			

		write(sum);
		;	
	}rof ;
  sum = sum^2;
  write(sum);
}</string>
         </property>
         <property name="overwriteMode">
          <bool>false</bool>
         </property>
         <property name="tabStopDistance">
          <double>20.000000000000000</double>
         </property>
         <property name="cursorWidth">
          <number>2</number>
         </property>
         <property name="backgroundVisible">
          <bool>false</bool>
         </property>
         <property name="centerOnScroll">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::DashLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="maximumSize">
        <size>
         <width>645</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Highlite Error</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Status:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuAnalyze">
    <property name="title">
     <string>Analyze</string>
    </property>
    <addaction name="actionShow_Tree"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnalyze"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionShow_Tree">
   <property name="text">
    <string>Show Tree</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
